stdlib/builtins.pyi

```diff
@@ -161,8 +161,10 @@ class classmethod(Generic[_T, _P, _R_co]):
         def __wrapped__(self) -> Callable[Concatenate[type[_T], _P], _R_co]: ...
 
 class type:
+    # object.__base__ is None. Otherwise, it would be a type.
     @property
-    def __base__(self) -> type: ...
+    def __base__(self) -> type | None: ...
+    # object.__bases__ is (). Otherwise, it would be a non-empty tuple of type(s).
```

Comment:
Not sure we need this comment? It doesn't contradict the below type annotation, and seems kinda obvious to me :)

```suggestion
```